### ISSUE TITLE
Port System.Diagnostics documentation for .NET 8.0 APIs

### ISSUE THREAD

**Issue by carlossanlop** [dotnet/runtime#88567]
Below is the list of APIs that still show up as undocumented in dotnet-api-docs and were introduced in .NET 8.0.

Full porting instructions can be found in the [main issue](https://github.com/dotnet/runtime/issues/88561).

This task needs to be finished before the RC2 snap (September 18th).

<details><summary>Old table (latest in comments)</summary>

| Summary  | Parameters | TypeParameters | ReturnValue | API                                                                                                                                                                                                                                                                                                                                                                                                                      |
|----------|------------|----------------|-------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.DiagnosticSource.StartActivity1(System.Diagnostics.Activity,0)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/DiagnosticSource.xml)                                                                                                                                                                                                                                   |
| Missing  | Missing    | Missing        | NA          | [M:System.Diagnostics.DiagnosticSource.StopActivity1(System.Diagnostics.Activity,0)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/DiagnosticSource.xml)                                                                                                                                                                                                                                    |
| Missing  | Missing    | Missing        | NA          | [M:System.Diagnostics.DiagnosticSource.Write1(System.String,0)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/DiagnosticSource.xml)                                                                                                                                                                                                                                                         |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.ProcessStartInfo.#ctor(System.String,System.Collections.Generic.IEnumerable{System.String})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/ProcessStartInfo.xml)                                                                                                                                                                                                      |
| Missing  | NA         | NA             | NA          | [P:System.Diagnostics.ProcessStartInfo.UseCredentialsForNetworkingOnly](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/ProcessStartInfo.xml)                                                                                                                                                                                                                                                 |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.Instrument.#ctor(System.Diagnostics.Metrics.Meter,System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Instrument.xml)                                                                              |
| Missing  | NA         | NA             | NA          | [P:System.Diagnostics.Metrics.Instrument.Tags](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Instrument.xml)                                                                                                                                                                                                                                                                        |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.Instrument1.#ctor(System.Diagnostics.Metrics.Meter,System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Instrument1.xml)                                                                            |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.InstrumentRecorder1.#ctor(System.Diagnostics.Metrics.Instrument)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/InstrumentRecorder1.xml)                                                                                                                                                                                                              |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.InstrumentRecorder1.#ctor(System.Object,System.String,System.String)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/InstrumentRecorder1.xml)                                                                                                                                                                                                          |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.InstrumentRecorder1.#ctor(System.Diagnostics.Metrics.Meter,System.String)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/InstrumentRecorder1.xml)                                                                                                                                                                                                     |
| Missing  | NA         | NA             | NA          | [M:System.Diagnostics.Metrics.InstrumentRecorder1.Dispose](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/InstrumentRecorder1.xml)                                                                                                                                                                                                                                                   |
| Missing  | Missing    | NA             | Missing     | [M:System.Diagnostics.Metrics.InstrumentRecorder1.GetMeasurements(System.Boolean)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/InstrumentRecorder1.xml)                                                                                                                                                                                                                           |
| Missing  | NA         | NA             | NA          | [P:System.Diagnostics.Metrics.InstrumentRecorder1.Instrument](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/InstrumentRecorder1.xml)                                                                                                                                                                                                                                                |
| Missing  | NA         | Missing        | NA          | [T:System.Diagnostics.Metrics.InstrumentRecorder1](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/InstrumentRecorder1.xml)                                                                                                                                                                                                                                                           |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.Meter.#ctor(System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}},System.Object)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                         |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.Meter.#ctor(System.Diagnostics.Metrics.MeterOptions)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                                                                                                                                        |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateCounter1(System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateHistogram1(System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                              |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateObservableCounter1(System.String,System.Func{0},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                       |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateObservableCounter1(System.String,System.Func{System.Collections.Generic.IEnumerable{System.Diagnostics.Metrics.Measurement{0}}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)       |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateObservableCounter1(System.String,System.Func{System.Diagnostics.Metrics.Measurement{0}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                               |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateObservableGauge1(System.String,System.Func{System.Diagnostics.Metrics.Measurement{0}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                 |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateObservableGauge1(System.String,System.Func{0},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                         |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateObservableGauge1(System.String,System.Func{System.Collections.Generic.IEnumerable{System.Diagnostics.Metrics.Measurement{0}}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)         |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateObservableUpDownCounter1(System.String,System.Func{0},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                 |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateObservableUpDownCounter1(System.String,System.Func{System.Diagnostics.Metrics.Measurement{0}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                         |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateObservableUpDownCounter1(System.String,System.Func{System.Collections.Generic.IEnumerable{System.Diagnostics.Metrics.Measurement{0}}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml) |
| Missing  | Missing    | Missing        | Missing     | [M:System.Diagnostics.Metrics.Meter.CreateUpDownCounter1(System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                          |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.Meter.Dispose(System.Boolean)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                                                                                                                                                               |
| Missing  | NA         | NA             | NA          | [P:System.Diagnostics.Metrics.Meter.Scope](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                                                                                                                                                                                 |
| Missing  | NA         | NA             | NA          | [P:System.Diagnostics.Metrics.Meter.Tags](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                                                                                                                                                                                  |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.MeterOptions.#ctor(System.String)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                    |
| Missing  | NA         | NA             | NA          | [T:System.Diagnostics.Metrics.MeterOptions](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                         |
| Missing  | NA         | NA             | NA          | [P:System.Diagnostics.Metrics.MeterOptions.Name](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                    |
| Missing  | NA         | NA             | NA          | [P:System.Diagnostics.Metrics.MeterOptions.Scope](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                   |
| Missing  | NA         | NA             | NA          | [P:System.Diagnostics.Metrics.MeterOptions.Tags](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                    |
| Missing  | NA         | NA             | NA          | [P:System.Diagnostics.Metrics.MeterOptions.Version](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                 |
| Missing  | Missing    | NA             | NA          | [M:System.Diagnostics.Metrics.ObservableInstrument1.#ctor(System.Diagnostics.Metrics.Meter,System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/ObservableInstrument1.xml)                                                        |

</details>

**Comment by carlossanlop** [comment]
It's mostly System.Diagnostics.Metrics, so I added that area label. But there are a few others from System.Diagnostics:

@dotnet/area-system-diagnostics-process  (owners of ProcessStartInfo).
@dotnet/area-system-diagnostics-tracing (owners of DiagnosticSource).

**Comment by tommcdon** [comment]
+ @tarekgh

**Comment by tarekgh** [comment]
I assigned the issue to myself.

**Comment by carlossanlop** [comment]
@tarekgh here's the most up-to-date list of APIs introduced in .NET 8 that still need docs. If you need help let me know and I can assist in documenting a few of them:

| API                                                                                                                                                                                                                                                                                                                                                                                                                      | Summary  | Parameters | TypeParameters | ReturnValue |
|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|------------|----------------|-------------|
| [M:System.Diagnostics.DiagnosticSource.StartActivity1(System.Diagnostics.Activity,0)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/DiagnosticSource.xml)                                                                                                                                                                                                                                   | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.DiagnosticSource.StopActivity1(System.Diagnostics.Activity,0)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/DiagnosticSource.xml)                                                                                                                                                                                                                                    | Missing  | Missing    | Missing        | NA          |
| [M:System.Diagnostics.DiagnosticSource.Write1(System.String,0)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/DiagnosticSource.xml)                                                                                                                                                                                                                                                         | Missing  | Missing    | Missing        | NA          |
| [M:System.Diagnostics.ProcessStartInfo.#ctor(System.String,System.Collections.Generic.IEnumerable{System.String})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/ProcessStartInfo.xml)                                                                                                                                                                                                      | Missing  | Missing    | NA             | NA          |
| [P:System.Diagnostics.ProcessStartInfo.UseCredentialsForNetworkingOnly](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics/ProcessStartInfo.xml)                                                                                                                                                                                                                                                 | Missing  | NA         | NA             | NA          |
| [M:System.Diagnostics.Metrics.IMeterFactory.Create(System.Diagnostics.Metrics.MeterOptions)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/IMeterFactory.xml)                                                                                                                                                                                                                       | Missing  | Missing    | NA             | Missing     |
| [T:System.Diagnostics.Metrics.IMeterFactory](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/IMeterFactory.xml)                                                                                                                                                                                                                                                                       | Missing  | NA         | NA             | NA          |
| [M:System.Diagnostics.Metrics.Instrument.#ctor(System.Diagnostics.Metrics.Meter,System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Instrument.xml)                                                                              | Missing  | Missing    | NA             | NA          |
| [P:System.Diagnostics.Metrics.Instrument.Tags](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Instrument.xml)                                                                                                                                                                                                                                                                        | Missing  | NA         | NA             | NA          |
| [M:System.Diagnostics.Metrics.Instrument1.#ctor(System.Diagnostics.Metrics.Meter,System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Instrument1.xml)                                                                            | Missing  | Missing    | NA             | NA          |
| [M:System.Diagnostics.Metrics.Meter.#ctor(System.Diagnostics.Metrics.MeterOptions)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                                                                                                                                        | Missing  | Missing    | NA             | NA          |
| [M:System.Diagnostics.Metrics.Meter.#ctor(System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}},System.Object)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                         | Missing  | Missing    | NA             | NA          |
| [M:System.Diagnostics.Metrics.Meter.CreateCounter1(System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateHistogram1(System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                              | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateObservableCounter1(System.String,System.Func{0},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                       | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateObservableCounter1(System.String,System.Func{System.Diagnostics.Metrics.Measurement{0}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                               | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateObservableCounter1(System.String,System.Func{System.Collections.Generic.IEnumerable{System.Diagnostics.Metrics.Measurement{0}}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)       | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateObservableGauge1(System.String,System.Func{System.Diagnostics.Metrics.Measurement{0}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                 | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateObservableGauge1(System.String,System.Func{0},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                         | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateObservableGauge1(System.String,System.Func{System.Collections.Generic.IEnumerable{System.Diagnostics.Metrics.Measurement{0}}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)         | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateObservableUpDownCounter1(System.String,System.Func{System.Collections.Generic.IEnumerable{System.Diagnostics.Metrics.Measurement{0}}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml) | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateObservableUpDownCounter1(System.String,System.Func{System.Diagnostics.Metrics.Measurement{0}},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                         | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateObservableUpDownCounter1(System.String,System.Func{0},System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                 | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.CreateUpDownCounter1(System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                          | Missing  | Missing    | Missing        | Missing     |
| [M:System.Diagnostics.Metrics.Meter.Dispose(System.Boolean)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                                                                                                                                                               | Missing  | Missing    | NA             | NA          |
| [P:System.Diagnostics.Metrics.Meter.Scope](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                                                                                                                                                                                 | Missing  | NA         | NA             | NA          |
| [P:System.Diagnostics.Metrics.Meter.Tags](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/Meter.xml)                                                                                                                                                                                                                                                                                  | Missing  | NA         | NA             | NA          |
| [M:System.Diagnostics.Metrics.MeterFactoryExtensions.Create(System.Diagnostics.Metrics.IMeterFactory,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterFactoryExtensions.xml)                                                           | Missing  | Missing    | NA             | Missing     |
| [T:System.Diagnostics.Metrics.MeterFactoryExtensions](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterFactoryExtensions.xml)                                                                                                                                                                                                                                                     | Missing  | NA         | NA             | NA          |
| [M:System.Diagnostics.Metrics.MeterOptions.#ctor(System.String)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                    | Missing  | Missing    | NA             | NA          |
| [T:System.Diagnostics.Metrics.MeterOptions](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                         | Missing  | NA         | NA             | NA          |
| [P:System.Diagnostics.Metrics.MeterOptions.Name](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                    | Missing  | NA         | NA             | NA          |
| [P:System.Diagnostics.Metrics.MeterOptions.Scope](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                   | Missing  | NA         | NA             | NA          |
| [P:System.Diagnostics.Metrics.MeterOptions.Tags](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                    | Missing  | NA         | NA             | NA          |
| [P:System.Diagnostics.Metrics.MeterOptions.Version](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/MeterOptions.xml)                                                                                                                                                                                                                                                                 | Missing  | NA         | NA             | NA          |
| [M:System.Diagnostics.Metrics.ObservableInstrument1.#ctor(System.Diagnostics.Metrics.Meter,System.String,System.String,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.Object}})](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Metrics/ObservableInstrument1.xml)                                                        | Missing  | Missing    | NA             | NA          |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.UIntPtr)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                         | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.UInt32)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                          | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.UInt16)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                          | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.String)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                          | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Single)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                          | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.SByte)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                           | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.IntPtr)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                          | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Boolean)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                         | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Int32)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                           | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Int64)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                           | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.UInt64)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                          | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Byte)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                            | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Byte[])~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                          | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Char)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                            | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Decimal)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                         | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.DateTime)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                        | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Double)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                          | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Enum)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                            | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Guid)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                            | Missing  | Missing    | NA             | Missing     |
| [M:System.Diagnostics.Tracing.EventSource.EventSourcePrimitive.op_Implicit(System.Int16)~System.Diagnostics.Tracing.EventSource.EventSourcePrimitive](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Diagnostics.Tracing/EventSource+EventSourcePrimitive.xml)                                                                                                                                           | Missing  | Missing    | NA             | Missing     |

**Comment by tarekgh** [comment]
@davmason could you please do the EventSource docs? Thanks!

**Comment by davmason** [comment]
@agocke it looks like all the EventSource ones are from your change, could you take a look?

**Comment by tommcdon** [comment]
> @agocke it looks like all the EventSource ones are from your change, could you take a look?

@agocke would you mind taking a look?  This issue is marked as release/8.0 blocking.

**Comment by tarekgh** [comment]
I addressed the Metrics docs. Someone needs to take care of the EventSource.

**Comment by tommcdon** [comment]
Looks like the remaining docs were completed on https://github.com/dotnet/dotnet-api-docs/pull/9288.  Please feel free to reactivate or create a new issue if there is anything missing.
